source/advanced/override-locales.rst

```diff
@@ -0,0 +1,46 @@
+Override GLPI locales
+---------------------
+
+GLPI uses the `gettext <https://www.gnu.org/software/gettext/>`_ tool to manage its translations. You can override the default.
+This is the preferred method rather than editing the original po files and risk losing your changes when updating GLPI.
+
+To do so, you need to create a file named ``filename.po`` in the ``files/_locales/core/`` directory.
+You can also create overrides for plugins with the same method in ``files/_locales/pluginkey/`` directory where ``pluginkey`` matches the plugin folder name.
+Depending on your GLPI configuration, the ``_locales`` directory can be located elsewhere (if you provided a ``local_define.php`` file with a ``GLPI_LOCAL_I18N_DIR`` constant).
+It's possible to have multiple files, to manage several languages for example.
```

Comment:
```suggestion

.. note::
   Depending on your GLPI configuration, the ``_locales`` directory can be located elsewhere (if you provided a ``local_define.php`` file with a ``GLPI_LOCAL_I18N_DIR`` constant).
   It's possible to have multiple files, to manage several languages for example.
```

Reply:
Either this, or split in two distinct paragraphs.

Reply:
ok for me